src/legacy/blueprint-parser.coffee

```diff
@@ -120,5 +126,4 @@ getAstSync = (code) ->
 module.exports = {
   parseApiaryBlueprintSync: getAstSync
   parse: getLocalAst
-  newVersionRegExp: NEW_VERSION_REGEXP # layout control control (editor)
```

Comment:
@honzajavorek 
Would there be any reason for us to want to leave the `RegExp` exported for `apiary`, or should we use `deckardcain` in `apiary`?